From 1f7aad701b78cbba821dc0bbde474fa9c4bc7313 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Fri, 29 Dec 2017 15:44:43 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3328: Add 200M for rkvdec

Add an intermediate opp so that rkvdec clock rate can be set to
an intermediate rate when temperature is above the trip point.

Change-Id: Ia94910185c708a501072c5da8aaebfcb206ad76b
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3328.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328.dtsi b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
index 806659f6eefc..60925a099b2f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
@@ -705,6 +705,10 @@
 			opp-hz = /bits/ 64 <100000000>;
 			opp-microvolt = <950000>;
 		};
+		opp-200000000 {
+			opp-hz = /bits/ 64 <200000000>;
+			opp-microvolt = <950000>;
+		};
 		opp-500000000 {
 			opp-hz = /bits/ 64 <500000000>;
 			opp-microvolt = <1050000>;
-- 
2.35.3

